tests/unittests/test_attribution_window.py

```diff
@@ -0,0 +1,32 @@
+import unittest
+import tap_facebook.__init__ as tap_facebook
+
+class TestAttributionWindow(unittest.TestCase):
```

Comment:
It's fine to leave in this unittest, but this should be a tap-tester case. We want to prove the error is thrown during discovery.

Reply:
@kspeer825 Added integration test for verifying error is thrown in discovery mode.